composer.json

```diff
@@ -6,7 +6,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.2|^8.0"
+        "php": ">=7.2 <=8.1"
```

Comment:
Is that useful since `config.platform.php` is at 7.3 ?

Reply:
I think composer still checks during installation

Reply:
I meant for 7.2 compatibility.